src/components/SingleEvent/EventCTA.js

```diff
@@ -97,11 +118,43 @@ class EventCTA extends Component {
 
     const myShare = calculateWinningShare(deposit, totalReg, numWent)
 
+    if (this.state.mode === take_all) {
+      return <WithdrawPayout address={address} amount={myShare} />
+    }
+    if (this.state.mode === donate) {
+      var all = myShare
+      var extra =
+        myShare -
+        toEthVal(deposit)
+          .toEth()
+          .toFixed(3)
+      var coffee = 0.01
```

Comment:
You can't just hard code 0.01 eth . You need to hit external price api to get the price .